requirements.yml

```diff
@@ -12,9 +12,11 @@ dependencies:
     - biopython
     - matminer
     - mdtraj
+    - mordred
```

Comment:
Can you add `mordred` and `mol2vec` to the list of optional dependencies in the docs?

Reply:
I fixed.